src/main/java/homeworks/fivesection/stringutils/StringUtils.java

```diff
@@ -0,0 +1,50 @@
+package homeworks.fivesection.stringutils;
+
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import java.util.regex.Pattern;
+
+public class StringUtils {
+
+    public static boolean isValidByOWASP(String email) {
+        String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@" +
```

Comment:
Lets do this simplier:
before dog, we have also symbols, also not less than 1, and with out some special characters; 
contains '@' domain- several symbols, not less than one, after it we should have dot, after is subdomain.